README.md

```diff
@@ -28,6 +28,17 @@ Usage
     * <kbd>CTRL-G</kbd><kbd>CTRL-R</kbd> for **R**emotes
     * <kbd>CTRL-G</kbd><kbd>CTRL-H</kbd> for commit **H**ashes
     * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> for **S**tashes
+  > :warning: You may have issues with these bindings in the following cases:
+  >
+  > * <kbd>CTRL-G</kbd><kbd>CTRL-B</kbd> will not work if
+  >   <kbd>CTRL-B</kbd> is used as the tmux prefix (bindkey)
+  > * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> will not work if flow control via
+  >   <kbd>CTRL-S</kbd> is enabled
```

Comment:
I'd prefer to mention that the terminal might get "frozen" upon hitting <kbd>CTRL-S</kbd>. Something like that maybe?

```suggestion
  > * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> will not work if flow control is enabled,
  >   <kbd>CTRL-S</kbd> will freeze the terminal instead
```

Reply:
Good idea.